test/instrumentations/next/app/src/app/layout.tsx

```diff
@@ -0,0 +1,23 @@
+import type { Metadata } from "next";
```

Comment:
Unless the test app will break by removing this, let's remove this as well as the following `tsx` files.

Reply:
Done